0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+
+# Mapping
+
+We use QGIS and Mergin Maps for our fish passage data management and analysis. This section will get you started with our mapping procedures.
+
+## QGIS
+
+* QGIS (Quantum Geographic Information System) is a free, open-source software that allows users to create, edit, visualize, analyze, and publish geospatial information.
+
+* Follow this link to the [QGIS Download Page](https://www.qgis.org/en/site/forusers/download.html) to install the latest version.
+
+* Here is some great info for getting up to speed with the [basics of QGIS](https://courses.spatialthoughts.com/introduction-to-qgis.html)
+
+
+## Mergin Maps
+
+[Mergin Maps](https://merginmaps.com/) is a geodata platform that allows us to collaborate with others by collecting data in the field and syncing to a shared QGIS project. Check out the [Mergin Maps documentation](https://merginmaps.com/docs/) for more information.
+
+### Set up 
+
+Set up Mergin Maps with QGIS on your laptop:
+
+  1. Follow the instruction to [sign up for a free mergin account](https://merginmaps.com/docs/setup/sign-up-to-mergin-maps/)
+  
+  2. Make sure you have the latest version of QGIS installed
+  
+  3. Install the [Mergin Maps plugin](https://merginmaps.com/docs/setup/install-mergin-maps-plugin-for-qgis/)
+  
+  4. Install the Mergin Maps app on your phone (we use this during field season)
+  
+  5. We will send you an invitation to join the `newgraph` Mergin workspace.
+  
+  
+Set up Mergin Maps on your phone, following the [instructions here](https://merginmaps.com/docs/setup/install-mobile-app/).
```

Comment:
This can be at the top of the section and can let users know to get the app at App store (for mac) and the android place for android phones (don't know the name of it).  They need to know that they need their username and password (or email - can't remember which one it needs).  We can refer to instructions but just getting the app and signing in is the meat and potatoes of what needs to happen.  

As per the email - QGIS and a laptop is not needed to use the maps

Reply:
already included.